### DISCUSSION TITLE
We are unable to provide this service. Tagged account

### DISCUSSION THREAD

**Discussion by hum4noidx** [community/community#62022] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I got this message:
"It appears your account may be based in a U.S.-sanctioned region. As a result, we are unable to provide private repository services and paid services for your account. GitHub has preserved, however, your access to [certain free services for public repositories](https://docs.github.com/en/github/site-policy/github-and-trade-controls#what-is-available-and-not-available). If your account has been flagged in error, and you are not located in or resident in a sanctioned region, please [file an appeal](https://docs.github.com/en/github/site-policy/github-and-trade-controls#how-is-github-ensuring-that-folks-not-living-in-andor-having-professional-links-to-the-sanctioned-countries-and-territories-still-have-access-or-ability-to-appeal). Please read about [GitHub and Trade Controls](https://docs.github.com/en/github/site-policy/github-and-trade-controls) for more information."

The problem is, that I never been in any of the sanctioned regions and my appeal still got no answer. I have a product release coming and can't access any of my organization repositories. Is it possible to speed up this process?

**Comment by hum4noidx** [comment]
@queenofcorgis Hi, can you please give any information about the tickets? 
#2267215
#2271628

**Reply by queenofcorgis** [reply]
Hi @hum4noidx, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
